pkg/apiserver/utils/binary_plan.go

```diff
@@ -554,13 +569,17 @@ func analyzeDurationNodes(nodes *simplejson.Json, operator operator, concurrency
 					}
 				}
 
+				actRowsStr := n.Get(ActRows).MustString()
+				if actRowsStr == "" {
+					continue
+				}
 				actRows, err = strconv.ParseFloat(n.Get(ActRows).MustString(), 64)
 				if err != nil {
 					return 0, err
 				}
 
 				taskCount := int(actRows * (1 - cacheHitRatio/100))
-
+				taskCount = taskCount / concurrency.joinConcurrency * concurrency.shuffleConcurrency * concurrency.tableConcurrency
```

Comment:
What is the actual meaning of these lines? How about renaming this to taskCountAvg or something more appropriate?